### PR TITLE
Add Python 3.6

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

**Comment by cjermain** [comment]
@basnijholt, thanks for starting to migrate this. I think that if you increment `number` to 1 in the `build` section of `meta.yaml` that should initiate a new build for Python 3.6. The Python 3.5 builds are successful, and the only difference is that a Python 3.6 build of 36_0 does not exist.

@melund or @ddale, can you add me as a manager to this feedstock? (I'm trying to update it so that I can release [pymeasure](https://github.com/ralph-group/pymeasure) on Python 3.6.)

**Comment by jakirkham** [comment]
It seems the problem is `pyvisa` is missing dependencies on Python 3.6, but that doesn't make a lot of sense looking at [its dependencies]( https://github.com/conda-forge/pyvisa-feedstock/blob/20a4a2c621634a97a4cf5d345756d6f94538f888/recipe/meta.yaml#L22-L25 ).

**Comment by cjermain** [comment]
Shouldn't `meta.yaml` line [L24](https://github.com/conda-forge/pyvisa-feedstock/blob/20a4a2c621634a97a4cf5d345756d6f94538f888/recipe/meta.yaml#L24)  be `- enum34 # [py<34]` instead of `- enum34 # [not (py35 or py34)]` to catch the Python 3.6 case?

**Comment by jakirkham** [comment]
Good eye! I had overlooked that. Indeed it should.

**Comment by basnijholt** [comment]
Great, thanks @cjermain, that fixed it :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ requirements:
   run:
     - python
     - setuptools
-    - enum34  # [not (py35 or py34)]
+    - enum34  # [py<34]
```
Comment (jakirkham) [inline recipe/meta.yaml:24]: Does this work ok with re-rendering? Had some issues with selectors like this in the past. Putting `py2k` might work better if it does have issues.

**Comment by cjermain** [comment]
Looks good! Thanks @basnijholt.

**Pull request merged**